src/ClassApplication/Controllers/ValuesController.cs

```diff
@@ -58,5 +58,40 @@ public async Task<ActionResult<string>> RemoveNoteFromClass(string classId, stri
             throw new NotImplementedException();
         }
 
+        /// <summary>
+        ///This call is used to delete a classroom
+        ///by classID
+        ///</summary>
+        ///<param name= "classId">ID of the classroom </param>
+        [HttpDelete("{classId}")]
+        public async Task<ActionResult<string>> deleteClass(string classId)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        ///This call is used to update the name and/or description of a class
+        ///by classId
+        ///</summary>
+        ///<param name="classId">Id of the classroom</param>
+        ///<param name="className">Name of classroom to set</param>
+        ///<param name="classDescription">Description of classroom to set</param>
+        [HttpPut("{classId}")]
+        public async Task<ActionResult<string>> updateClass(string classId, string className, string classDescription)

```

Comment:
nit: You have your method names start with either upper case or lower case letters. Check out line 92 and please make everything consistent